From 25a7a8006a3d4d6e08fe5518acfdefa4fe8e1acd Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Florian=20M=C3=BCllner?= <fmuellner@gnome.org>
Date: Tue, 27 Aug 2019 18:11:33 +0200
Subject: [PATCH 045/152] gnome-extensions: Translate help command

The quotes and COMMAND string aren't part of the syntax, so they
should be translated.

https://gitlab.gnome.org/GNOME/gnome-shell/issues/1538
---
 src/extensions-tool/main.c | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/src/extensions-tool/main.c b/src/extensions-tool/main.c
index dafca2123..7cc83de95 100644
--- a/src/extensions-tool/main.c
+++ b/src/extensions-tool/main.c
@@ -236,6 +236,10 @@ handle_version (int argc, char *argv[], gboolean do_help)
 static void
 usage (void)
 {
+  g_autofree char *help_command = NULL;
+
+  help_command = g_strdup_printf ("gnome-extensions help %s", _("COMMAND"));
+
   g_printerr ("%s\n", _("Usage:"));
   g_printerr ("  gnome-extensions %s %s\n", _("COMMAND"), _("[ARGS…]"));
   g_printerr ("\n");
@@ -254,7 +258,7 @@ usage (void)
   g_printerr ("  pack      %s\n", _("Package extension"));
   g_printerr ("  install   %s\n", _("Install extension bundle"));
   g_printerr ("\n");
-  g_printerr (_("Use %s to get detailed help.\n"), "“gnome-extensions help COMMAND”");
+  g_printerr (_("Use “%s” to get detailed help.\n"), help_command);
 }
 
 int
-- 
2.23.0

